cmd/arc/services/blocktx.go

```diff
@@ -2,14 +2,13 @@ package cmd
 
 import (
 	"fmt"
+	natscore "github.com/bitcoin-sv/arc/internal/message_queue/nats/client/nats_core"
+	natsjetstream "github.com/bitcoin-sv/arc/internal/message_queue/nats/client/nats_jetstream"
+	natsconnection "github.com/bitcoin-sv/arc/internal/message_queue/nats/nats_connection"
```

Comment:
No aliases needed here, as this rule has been excluded

Reply:
Removed